pkg/ccl/backupccl/restore_online_test.go

```diff
@@ -82,6 +82,69 @@ func TestOnlineRestoreBasic(t *testing.T) {
 	rSQLDB.CheckQueryResults(t, createStmt, createStmtRes)
 }
 
+func TestOnlineRestoreStatementResult(t *testing.T) {
+	defer leaktest.AfterTest(t)()
+	defer log.Scope(t).Close(t)
+	defer nodelocal.ReplaceNodeLocalForTesting(t.TempDir())()
+
+	const numAccounts = 1
+	_, sqlDB, _, cleanupFn := backupRestoreTestSetupWithParams(t, singleNode, numAccounts, InitManualReplication, base.TestClusterArgs{
+		ServerArgs: base.TestServerArgs{
+			DefaultTestTenant: base.TestIsSpecificToStorageLayerAndNeedsASystemTenant,
+		},
+	})
+	defer cleanupFn()
+
+	if err := backuptestutils.VerifyBackupRestoreStatementResult(
```

Comment:
nit: non need to verify the backup statement result. we already have tests that do this.